src/.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Let's remove the `.vscode` files. They're meant for developer-specific configs on local machines, so shouldn't be part of the repository.

Bonus: Add the `.vscode` folder to the `.gitignore` file. This should prevent it from being staged and committed in the future as well! :)